#! /bin/sh /usr/share/dpatch/dpatch-run
## 03filestatus-amulecmd.dpatch by Julien Delange <julien@gunnm.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Can't control file status with amulecmd due to a bug in hash checking (#360135)

@DPATCH@
--- ../src/TextClient.cpp.orig	2006-04-19 03:53:59.000000000 +0200
+++ ../src/TextClient.cpp	2006-04-19 03:56:23.000000000 +0200
@@ -281,7 +281,7 @@
 				}
 			} else {
 				CMD4Hash hash;
-				if (not hash.Decode(args)) {
+				if ( hash.Decode(args.Trim(false).Trim(true)) ) {
 					if (!hash.IsEmpty()) {
 						switch(CmdId) {
 							case CMD_ID_PAUSE:
